sphinx/tutorial/cylc/scheduling/advanced-scheduling.rst

```diff
@@ -0,0 +1,73 @@
+Advanced Scheduling
+===================
+
+In this section we will quickly run through some of the more advanced features
+of cylc's scheduling logic.
+
+
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+
+.. _tutorial-qualifiers:
+
+Qualifiers
+----------
+
+So far we have written dependencies like ``foo => bar``. This is, in-fact,
+shorthand for ``foo:succeed => bar``, it means that the task ``bar`` will run
+once ``foo`` has finished successfully. If ``foo`` were to fail then ``bar``
+would not run. We will talk more about these :term:`task states <task state>`
+in the TODO:Runtime_Section.
+
+We would refer to the ``:succeed`` as a :term:`qualifier`. There are qualifiers
+for different :term:`task states <task state>` e.g:
+
+``:start``
+    When the task has started running.
+``:fail``
+    Occurs when the task finishes if it fails (non-zero return code).
+``:finish``
+    Occurs when the task has stopped running irrespective of whether it has
+    succeeded or failed.
+
+It is also possible to create your own custom :term:`qualifiers <qualifier>`
+to handle events within your code (custom outputs).
+
+*For more information see the* `cylc user guide`_.
+
+
+Clock Triggers
+--------------
+
+In cylc :term:`cycle points <cycle point>` are just labels. When using datetime
+cycling, clock triggers can be used to force cycles to wait for a particular
```

Comment:
> ...[date-time] cycle points are just labels.

I think this is an important point to stress. Maybe follow up with something along the lines of: 

> "In other words, a cycle point labelled with a date-time will be triggered as soon as its dependencies have met, and it will not wait for the actual datetime to be reached. However, this can be achieved with..."

[not sure of the exact wording here, but words to this effect?]